Exonoesis/Helping-Hand
# Spellchecking for Asset Loading

This new feature adds suggestion log messages for files or folders that were not found by the asset server in the event that there are non-standard characters that look similar to standard characters in file names, for example.

To test locally:
1. Navigate to `src/visuals/map.rs`
2. Change the loaded asset from `map/hh_world.ldtk` to `map/hh_asdfa.ldtk`
3. Run the game.
4. Observe in the terminal that the logging suggests `map/hh_world.ldtk` in its full path.
